android/src/main/java/com/reactnativestripesdk/PaymentSheetFragment.kt

```diff
@@ -191,7 +193,11 @@ class PaymentSheetFragment(
       .allowsDelayedPaymentMethods(allowsDelayedPaymentMethods ?: false)
       .defaultBillingDetails(defaultBillingDetails)
       .customer(
-        if (customerId.isNotEmpty() && customerEphemeralKeySecret.isNotEmpty()) PaymentSheet.CustomerConfiguration(
+        if (customerId.isNotEmpty() && customerSessionClientSecret.isNotEmpty()) PaymentSheet.CustomerConfiguration.createWithCustomerSession(
+          id = customerId,
+          clientSecret = customerSessionClientSecret
+        )
+        else if (customerId.isNotEmpty() && customerEphemeralKeySecret.isNotEmpty()) PaymentSheet.CustomerConfiguration(
```

Comment:
do we want to add error handling if they provide both `customerSessionClientSecret` and `customerEphemeralKeySecret`?

Reply:
Updated 👍